HttpTrigger1/main.py

```diff
@@ -6,13 +6,21 @@
 
 
 from .models import Label, Participant, Meeting, create_tables, database_location
+from .auth import *
 
 ACCESS_TOKEN = os.environ["GITHUB_ACCESS_TOKEN"]
 REPO_FULL_NAME = "ElGarash/Jitsi-Meetings-Dashboard"
 TARGET_BRANCH = "gh-pages"
 
 
-def main(req: func.HttpRequest, context: func.Context) -> func.HttpResponse:
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    try:
+        token = get_token_auth_header(req)
+        payload = verify_decode_jwt(token)
+        check_permissions(PERMISSION, payload)
+    except AuthError as e:
+        return func.HttpResponse(f"{e.message}", status_code=e.status_code)
```

Comment:
You don't need an f-string here.
```suggestion
        return func.HttpResponse(e.message, status_code=e.status_code)
```